src/krb5/_change_password.pyi

```diff
@@ -0,0 +1,122 @@
+import typing
+
+from krb5._ccache import CCache
+from krb5._context import Context
+from krb5._creds import Creds
+from krb5._principal import Principal
+
+def change_password(
+    context: Context,
+    creds: Creds,
+    newpw: bytes,
+) -> typing.Tuple[int, bytes, bytes]:
+    """Set a password for the specified credentials owner.
+
+
+    This function implements the set password operation of ``RFC 3244``,
+    for interoperability with Microsoft Windows implementations.
+    It uses the credentials `creds` to change the password to `newpw`.
+
+    Note: obtain the `creds` using `get_init_creds_password()` with
+    in_tkt_service set to "kadmin/changepw".
+
+    Args:
+        context: Krb5 context.
+        creds: Credentials for kadmin/changepw service.
+        newpw: New password.
+        change_password_for: `None` or the principal to set the password for.
```

Comment:
Seems like this isn't an arg and might just be a copy/paste from `set_password`. The docstring might also benefit from a futher explanation as to why you might want to use `change_password` over one of the `set_password` variants.

Reply:
typo indeed; anyway `change_password()` is [deprecated](https://github.com/jborean93/pykrb5/pull/50#discussion_r1730910751) and dropped